### PR TITLE
implemented vertical temperature scale

### PR DESCRIPTION
**Pull request by Kevinwochan**

### CODE REVIEW

**Comment by Kevinwochan** [comment]
Positioning is very hacky, i'm not sure how to fix that without gridding the page.

File: src/components/Map/TemperatureScale.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    { value: 1, description: "13\u2103", color: '#0245ff' },
+                    { value: 1, description: "12\u2103", color: '#33007f' },
+                    { value: 1, description: "11\u2103", color: '#800080' },
+                    { value: 1, description: "10\u2103", color: '#805880' }
+                ]}
+            />
+        </ChartBarWrapper>);
+}
+
+export default TemperatureScale;
```
Comment (r007) [inline src/components/Map/TemperatureScale.js:43]: Can you add a newline here, please?

File: src/components/Map/VerticalStackedBar.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+};
+
+VeritcalBar.defaultProps = {
+    width: 30,
+    showTextWithValue: true,
+    outlineHeight: 0,
+    outlineColor: "black",
+    fontColor: "white",
+    id: "vsbar"
+};
```
Comment (r007) [inline src/components/Map/VerticalStackedBar.js:188]: Can you add a newline here, please?

**Comment by r007** [comment]
I need to try this on localhost...

**Comment by Kevinwochan** [comment]
@r007 
I'll add those newlines in now.

> I need to try this on localhost...
hopefully i'll fix this with Tom on Monday so all PRs will create an amplify instance

**Comment by Kevinwochan** [comment]
@r007 AWS Amplify doesnt seem to be playing nicely with this PR, closing and making a new one.

**Pull request closed without merging**